docs/marketplace/dev/feature-integration-guides/202106.0/merchant-portal-core-feature-integration.md

```diff
@@ -136,6 +140,64 @@ class SecurityDependencyProvider extends SprykerSecurityDependencyProvider
 }
 ```
 
+**src/Pyz/Zed/UserPasswordReset/UserPasswordResetDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\UserPasswordReset;
+
+use Spryker\Zed\UserPasswordReset\UserPasswordResetDependencyProvider as SprykerUserPasswordResetDependencyProvider;
+use Spryker\Zed\MerchantUserPasswordResetMail\Communication\Plugin\UserPasswordReset\MailMerchantUserPasswordResetRequestStrategyPlugin;
+
+class UserPasswordResetDependencyProvider extends SprykerUserPasswordResetDependencyProvider
+{
+    /**
+     * @return \Spryker\Zed\UserPasswordResetExtension\Dependency\Plugin\UserPasswordResetRequestStrategyPluginInterface[]
+     */
+    public function getUserPasswordResetRequestStrategyPlugins(): array
+    {
+        return [
+            new MailMerchantUserPasswordResetRequestStrategyPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Zed/Mail/MailDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Mail;
+
+use Spryker\Zed\Mail\MailDependencyProvider as SprykerMailDependencyProvider;
+use Spryker\Zed\MerchantUserPasswordResetMail\Communication\Plugin\Mail\MerchantUserPasswordResetMailTypePlugin;
+
+class MailDependencyProvider extends SprykerMailDependencyProvider
+{
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    public function provideBusinessLayerDependencies(Container $container)
+    {
+        $container = parent::provideBusinessLayerDependencies($container);
+        $container->extend(static::MAIL_TYPE_COLLECTION, function (MailTypeCollectionAddInterface $mailCollection) {
+            $mailCollection
+            ->add(new MerchantUserPasswordResetMailTypePlugin());
+            
+            return $mailCollection;
+        });
+        
+        return $container;
+    }
+}
+```
+
+
+
 Open access to the Merchant Portal login page by default:
```

Comment:
Should be part of `Set up Configuration` section
https://spryker.atlassian.net/wiki/spaces/DOCS/pages/478118714/Feature+integration+guides